src/composables/useRenderer/const.ts

```diff
@@ -12,6 +12,10 @@ export const rendererPresets = {
       type: PCFSoftShadowMap,
     },
   },
+  flat: {
+    toneMapping: NoToneMapping,
```

Comment:
I added it here, but is not working at all when you set the preset to `flat` 

@Tinoooo should we tackle it here or in a separate PR?